core/include/tee/tee_svc.h

```diff
@@ -52,7 +52,15 @@ uint32_t syscall_dummy_7args(unsigned long a1, unsigned long a2,
 
 uint32_t syscall_nocall(void);
 
-TEE_Result syscall_get_property(unsigned long prop, void *buf, size_t blen);
+unsigned long syscall_get_number_properties(unsigned long propSet);
+TEE_Result syscall_get_property(unsigned long prop_set,
+				void *name, unsigned long name_len,
```

Comment:
Suppose you would drop the `name` and `name_len` arguments and only use `index`, then the  `syscall_get_property_name()` function would become  `syscall_property_name_to_index()`.